cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,94 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('should test pagination', () => {
+    cy.get('.left-pannel').click();
+  });
+
+  it('should test rows count selection', () => {
+    cy.get('.col-md-6').should('be.visible').click();
+  });
+
+  it('should add a new worker', () => {
+    cy.get('#addNewRecordButton').click();
+    const firstName = 'Kierra';
+    const lastName = 'Gentry';
+    const userEmail = 'kierra@example.com';
+    const age = '29';
+    const salary = '2000';
+    const department = 'Legal';
+    cy.get('#firstName').type(firstName);
+    cy.get('#lastName').type(lastName);
+    cy.get('#userEmail').type(userEmail);
+    cy.get('#age').type(age);
+    cy.get('#salary').type(salary);
+    cy.get('#department').type(department);
+    cy.get('#submit').click();
+  });
+
+  it('should find and edit a worker', () => {
+    cy.contains('Kierra')
```

Comment:
Test 'should find and edit a worker' assumes the worker 'Kierra' is present in the table. Add assertion to confirm worker 'Kierra' is present before attempting to edit.